### DISCUSSION TITLE
Allow replying to messages by right-clicking usernames

### DISCUSSION THREAD

**Discussion by aidenwallis** [Chatterino/chatterino2#3934] [Ideas] [answered]
It's common to right click usernames to mention someone when sending a message - it feels natural to have the ability to "shit+right click" someones username to send a reply, instead of having to right-click on message content and clicking "Reply to message".

It would save an extra click and adds less changes to existing workflows.

**Comment by Felanbird** [comment, marked as answer]
Added in #4424
